pkg/cli/admin/prune/deployments/resolvers.go

```diff
@@ -129,7 +129,9 @@ func (o *perDeploymentResolver) Resolve() ([]metav1.Object, error) {
 			case *corev1.ReplicationController:
 				status = appsutil.DeploymentStatusFor(v)
 			case *kappsv1.ReplicaSet:
-				status = appsutil.DeploymentStatusFor(v)
+				if v.Status.Replicas == 0 {
+					status = "Complete"
```

Comment:
what if this is the latest revision rs with just 0 replicas? We probably don't want to prune this one.

Reply:
There only seemed to be two status fields on ReplicaSets, so determining complete vs failed, like for ReplicationControllers, seems much more difficult. I'm open to suggestions on this one.